src/utils.js

```diff
@@ -1,9 +1,29 @@
 import * as Errors from 'bitgo/dist/src/errors';
 
-const userKeyPaths = ['/0/0', '/0'];
+import * as utxolib from '@bitgo/utxo-lib';
 
-function withCustomKeyPath(params, userKeyPath) {
-  return Object.assign({}, params, { userKeyPath });
+import { AbstractUtxoCoin } from 'bitgo/dist/src/v2/coins';
+
+function sanitizeKeys(keys) {
+  return keys.map((k) => {
+    if (!(k instanceof utxolib.HDNode)) {
+      throw new Error(`unexpected key`);
+    }
+
+    return k.neutered().toBase58();
+  });
+}
+
+export async function getRecoveryDebugInfo(baseCoin, recoveryParams) {
+  if (!(baseCoin instanceof AbstractUtxoCoin)) {
+    // TODO support more coins
+    throw new Error('unsupported coin');
```

Comment:
We should use the coin-config as a point of reference, as we support different coins for testnet and mainnet as well (due to some testnet block explorers being unavailable) 

See https://github.com/BitGo/wallet-recovery-wizard/blob/1d29e4812adeff2119f14e7595f46266ccba1da5/src/constants/coin-config.js#L133

Reply:
In this case though we only worry about the coins that `getRecoveryDebugInfo()` method works for.

Since I can't guarantee that the result of `initiateRecovery` works with other coins I narrowed it down to utxo-like coins. It works for all utxo coins though.